recipes/multicollections/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "multicollections" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/foamlib-{{ version }}.tar.gz
+  sha256: 44fe33495a49aeef4c9bec16037d80d9ea8eddc61e1d0047777e9f15780206ee
+
+build:
+  skip: true  # [py<37]
```

Comment:
You should remove this and make this package `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python